File: python/01_hello.py
"""Smallest possible job: confirm the environment is what you think it is."""
import os
import platform
import sys

print(f"host          : {platform.node()}")
print(f"python        : {sys.version.split()[0]}")
print(f"job id        : {os.environ.get('SLURM_JOB_ID')}")
print(f"cpus per task : {os.environ.get('SLURM_CPUS_PER_TASK')}")

File: julia/01_hello.jl
# Smallest possible job: confirm the environment is what you think it is.
println("host          : ", gethostname())
println("julia         : ", VERSION)
println("job id        : ", get(ENV, "SLURM_JOB_ID", "not in a job"))
println("threads       : ", Threads.nthreads())
println("cpus per task : ", get(ENV, "SLURM_CPUS_PER_TASK", "unset"))
